public/index.html

```diff
@@ -0,0 +1,741 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Brand Marketing App</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="description" content="Project Week One">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300,100,700' rel='stylesheet' type='text/css'>
```

Comment:
`href` attributes need to use `"` (double quotes) in order to be valid. I'd use them on `rel` and `type` as well, but for `href` it's required.